### PR TITLE
First commit with git!

### PR DESCRIPTION
**Pull request by Sbruk**
Update caption for TsnC5 in **GitHub** thanks to _Git_! Issue is #1 

@kesslerb Please approve.

### CODE REVIEW

File: _posts/0000-01-03-TsnC5.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![riddlocat](https://octodex.github.com/images/riddlocat.png)
 {: .center}
 
-Caption here
+Hello GitHub !
```
Comment (Sbruk) [inline _posts/0000-01-03-TsnC5.md]: @alueib Looks good, nah?

Comment (alueib) [reply _posts/0000-01-03-TsnC5.md]: amaaaaazing

Comment (Sbruk) [reply _posts/0000-01-03-TsnC5.md]: 😍

Comment (mayaydineib) [reply _posts/0000-01-03-TsnC5.md]: Looks nice

Comment (mayaydineib) [reply _posts/0000-01-03-TsnC5.md]: review done

**Review by kesslerb — APPROVED** [review]
Nice!

**Review by mayaydineib — APPROVED** [review]
ok

**Pull request merged**